Error in threading plugin when compiling on windows

I got an error in the threading plugin:

plugins\threading\src\vrthreadgroup.cpp(61): error C4716: 'MinVR::VRThreadGroup::waitForComplete' : must return a value

Ben

--- a/plugins/Threading/src/VRThreadGroup.cpp
+++ b/plugins/Threading/src/VRThreadGroup.cpp
@@ -51,7 +51,7 @@ void VRThreadGroup::completeAction() {
 	actionCompletedMutex.unlock();
 }
 
-VRRenderThreadAction VRThreadGroup::waitForComplete() {
+void VRThreadGroup::waitForComplete() {
 	// Waits until all the threads are completed
 	UniqueMutexLock completedActionLock(actionCompletedMutex);
 	while (numThreadsCompleted < numThreads) {

--- a/plugins/Threading/src/VRThreadGroup.h
+++ b/plugins/Threading/src/VRThreadGroup.h
@@ -49,7 +49,7 @@ public:
 	void completeAction();
 
 	/// Waits for all threads to be completed
-	VRRenderThreadAction waitForComplete();
+	void waitForComplete();
 
 	VRRenderHandler* getRenderHandler() const {
 		return renderHandler;

